src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.cs

```diff
@@ -68,6 +69,12 @@ public LibGpiodDriver(int gpioChip = 0)
             }
         }
 
+        [Obsolete("Use Board.CreateGpioController instead")]
```

Comment:
Comment in general for all the Gpio Driver Obsoletes: I don't think that we should go through this path into forcing people to use Board abstraction. One of the features we have in our design is to support people to create new GpioDrivers that internally use 2 or more drivers for cases where you are using pins from different providers. Doing that via the Board now would add an extra layer of complexity that wouldn't be desired.